src/scripts/modules/media/header/header-template.html

```diff
@@ -35,9 +35,9 @@ <h2>
       <div class="downloads">
         {{#or hasDownloads buyLink}}
           {{#is mediaType 'application/vnd.org.cnx.collection'}}
-            <button class="btn" tabindex="0" type="submit" data-l10n-id="textbook-view-btn-get-this-book">Get This Book!</button>
+            <button class="btn getContent" tabindex="0" type="submit" data-l10n-id="textbook-view-btn-get-this-book" aria-role="dialoge">Get This Book!</button>
           {{else}}
-            <button class="btn" tabindex="0" type="submit" data-l10n-id="textbook-view-btn-get-this-page">Get This Page!</button>
+            <button class="btn getContent" tabindex="0" type="submit" data-l10n-id="textbook-view-btn-get-this-page" aria-role="dialoge">Get This Page!</button>
```

Comment:
Should this attribute be `aria-role="dialog"` ?

Reply:
Also note the spelling of 'dialog'.